control/controlclient/status.go

```diff
@@ -76,33 +23,23 @@ type Status struct {
 	// URL, if non-empty, is the interactive URL to visit to finish logging in.
 	URL string
 
+	// LoginFinished, if true, indicates that the login process has
+	// successfully completed and the client is in the authenticated state.
+	LoginFinished bool
```

Comment:
can you say specifically what this means more? that we've done at least one successful serveRegister?